src/components/Dashboard/TalkDashboard.vue

```diff
@@ -0,0 +1,226 @@
+
+import { useStore } from '../../composables/useStore.js'
+import { CONVERSATION } from '../../constants.ts'
+import { useTalkDashboardStore } from '../../stores/talkdashboard.ts'
+import type { Conversation, DashboardEventRoom } from '../../types/index.ts'
+import { filterConversation } from '../../utils/conversation.ts'
+
+const FIVE_MINUTES = 5 * 60 * 1000 // 5 minutes
+const store = useStore()
+const router = useRouter()
+const talkDashboardStore = useTalkDashboardStore()
+
+const eventRooms = ref<DashboardEventRoom[]>([])
+const loading = ref(false)
+const fetchEventRoomsInterval = ref<ReturnType<typeof setInterval> | null>(null)
+
+onMounted(async () => {
+	fetchEventRooms()
+	fetchEventRoomsInterval.value = setInterval(fetchEventRooms, FIVE_MINUTES)
+})
+
+onBeforeUnmount(() => {
+	if (fetchEventRoomsInterval.value) {
+		clearInterval(fetchEventRoomsInterval.value)
+	}
+})
+
+const conversationsList = computed(() => store.getters.conversationsList)
+const conversationsInitialised = computed(() => store.getters.conversationsInitialised)
+const filteredConversations = computed(() => conversationsList.value?.filter((conversation : Conversation) => {
+	return filterConversation(conversation, ['mentions'])
+}))
+
+/**
+ * Fetches the event rooms
+ */
+async function fetchEventRooms() {
+	try {
+		loading.value = true
+		eventRooms.value = await talkDashboardStore.fetchDashboardEventRooms() ?? []
+		loading.value = false
+	} catch (error) {
+		console.error('Error fetching event rooms:', error)
+		showError(t('spreed', 'Error fetching event rooms'))
+	}
+}
+
+/**
+ * Creates a new group conversation and navigates to the conversation page.
+ */
+async function startMeeting() {
+	try {
+		const conversation = await store.dispatch('createGroupConversation', {
+			roomName: t('spreed', 'Meeting'), // NOTE: it uses user language statically
+			roomType: CONVERSATION.TYPE.PUBLIC,
+		})
+		router.push({
+			name: 'conversation',
+			params: { token: conversation.token },
+			hash: '#direct-call',
+		})
+	} catch (error) {
+		console.error('Error creating conversation:', error)
+		showError(t('spreed', 'Error creating a meeting'))
+	}
+}
+</script>
+<template>
+	<div class="talk-dashboard-wrapper">
+		<div class="talk-dashboard__header">
+			{{ t('spreed', 'Talk home') }}
+		</div>
+		<div class="talk-dashboard__title">
+			<span class="title">{{ t('spreed', 'Upcoming meetings') }}</span>
+			<NcButton type="primary"
+				@click="startMeeting">
+				<template #icon>
+					<IconVideo />
+				</template>
+				{{ t('spreed', 'Start meeting now') }}
+			</NcButton>
+		</div>
+		<div v-if="!loading && eventRooms.length > 0" class="talk-dashboard__event-cards">
+			<EventCard v-for="eventRoom in eventRooms"
+				:key="eventRoom.eventLink"
+				:event-room="eventRoom"
+				class="talk-dashboard__event-card" />
+		</div>
+		<div v-else-if="loading"
+			class="event-cards__loading-placeholder">
+			<LoadingPlaceholder type="event-cards" />
+		</div>
```

Comment:
```suggestion
		<LoadingPlaceholder v-else-if="loading"
			class="event-cards__loading-placeholder"
			type="event-cards" />
```

Make sure placeholder is horizontally scrollable on small screen